src/pages/new-blog/NewBlogPage.jsx

```diff
@@ -0,0 +1,114 @@
+import './NewBlogPage.css';
+
+import CalculateReadTime from "../../helpers/calculateReadTime.js";
+
+import {useState} from "react";
+import {useForm} from 'react-hook-form';
+import {Link} from "react-router-dom";
+import axios from 'axios';
+
+import TextFormControl from "../../components/form-controls/text-form-control/TextFormControl.jsx";
+import TextareaFormControl from "../../components/form-controls/textarea-form-control/TextareaFormControl.jsx";
+import Button from "../../components/button/Button.jsx"
+
+function NewBlogPage() {
+  const {
+    handleSubmit,
+    formState: {
+      errors
+    },
+    register
+  } = useForm();
+  const [createdBlogId, setCreatedBlogId] = useState('');
+  const [error, setError] = useState('');
+
+  const APIProjectIDHeader = {'Novi-Education-Project-Id': 'ec0bf4cc-4e94-4807-8041-d95b0731722b'};
+
+  async function handleFormSubmit(data) {
+    const newBlog = {
+      ...data,
+      shares: 0,
+      comments: 0,
+      created: new Date().toISOString(),
+      readTime: CalculateReadTime(data.content)
+    }
+
+    await postBlog(newBlog);
```

Comment:
Alles wat in de declaratie van `function postBlog(blog)` staat kun je gewoon hier schrijven! 

```
async function handleFormSubmit(data) {
    const newBlog = {}
    
    try {
      const response = await axios.post('/api/blogs', blog, {
        headers: {
          ...APIProjectIDHeader
        }
      }); // ... etc.
```